airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -69,31 +89,48 @@ def poke(self, context):
 
         self.log.info('SQSSensor checking for message on queue: %s', self.sqs_queue)
 
-        messages = sqs_conn.receive_message(
-            QueueUrl=self.sqs_queue,
-            MaxNumberOfMessages=self.max_messages,
-            WaitTimeSeconds=self.wait_time_seconds,
-        )
+        receive_message_kwargs = {
+            'QueueUrl': self.sqs_queue,
+            'MaxNumberOfMessages': self.max_messages,
+            'WaitTimeSeconds': self.wait_time_seconds,
+        }
+        if self.visibility_timeout is not None:
+            receive_message_kwargs['VisibilityTimeout'] = self.visibility_timeout
+
+        response = sqs_conn.receive_message(**receive_message_kwargs)
+
+        if "Messages" not in response:
+            return False
 
-        self.log.info("received message %s", str(messages))
+        messages = response['Messages']
+        num_messages = len(messages)
+        self.log.info("received %s messages", str(num_messages))
```

Comment:
```suggestion
        self.log.info("received %d messages", num_messages)
```

The string conversion is redundant since the logger can accept int directly; passing the int also ensures the most context is retained, which can be useful for people using custom log handlers and formatters. (I know this was done in existing code; it is wrong.)

Same for other logs below.

Reply:
Thanks for the feedback, I didn't know about this. I'll get them all fixed up

Reply:
Done